### ISSUE TITLE
Unhandled Exception: Microsoft.DotNet.ProjectModel.FileFormatException: unknown keyword platform

### ISSUE THREAD

**Issue by Smilefounder** [aspnet/cli-samples#48]
Got an exception below:

**Environment:**
D:\Workspace\GitHub\cli-samples\HelloWeb>dotnet --version
.NET Command Line Tools (1.0.0-beta-001598)

Product Information:
 Version:     1.0.0-beta-001598
 Commit Sha:  7582649f88

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 Runtime Id:  win10-x64

**Error:**
```
D:\Workspace\GitHub\cli-samples\HelloWeb>dotnet store

Unhandled Exception: Microsoft.DotNet.ProjectModel.FileFormatException: unknown keyword platform ---> System.InvalidOperationException: unknown keyword platform
   at Microsoft.DotNet.ProjectModel.Graph.LibraryDependencyType.Parse(String keyword)
   at Microsoft.DotNet.ProjectModel.ProjectReader.PopulateDependencies(String projectPath, IList`1 results, JsonObject settings, String propertyName, Boolean isGacOrFrameworkReference)
   at Microsoft.DotNet.ProjectModel.ProjectReader.ReadProject(Stream stream, String projectName, String projectPath, ICollection`1 diagnostics, ProjectReaderSettings settings)
   at Microsoft.DotNet.ProjectModel.ProjectReader.TryGetProject(String path, Project& project, ICollection`1 diagnostics, ProjectReaderSettings settings)
   --- End of inner exception stack trace ---
   at Microsoft.DotNet.ProjectModel.ProjectReader.TryGetProject(String path, Project& project, ICollection`1 diagnostics, ProjectReaderSettings settings)
   at Microsoft.DotNet.ProjectModel.ProjectContextBuilder.ResolveProject(String projectDirectory)
   at Microsoft.DotNet.ProjectModel.ProjectContextBuilder.EnsureProjectLoaded()
   at Microsoft.DotNet.ProjectModel.ProjectContextBuilder.Build()
   at Microsoft.DotNet.Cli.Utils.CommandResolver.ResolveFromProjectDependencies(String commandName, IEnumerable`1 args, NuGetFramework framework, String configuration)
   at Microsoft.DotNet.Cli.Utils.CommandResolver.TryResolveCommandSpec(String commandName, IEnumerable`1 args, NuGetFramework framework, String configuration)
   at Microsoft.DotNet.Cli.Utils.Command.Create(String commandName, IEnumerable`1 args, NuGetFramework framework, String configuration)
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args)
   at Microsoft.DotNet.Cli.Program.Main(String[] args)

D:\Workspace\GitHub\cli-samples\HelloWeb>
```

**Comment by davidfowl** [comment]
Follow the instructions on the readme to get a new CLI. https://github.com/aspnet/cli-samples/blob/master/README.md#how-to-run

**Comment by Smilefounder** [comment]
Hi @davidfowl 
I did exactly the same to the instruction from https://github.com/aspnet/cli-samples/blob/master/README.md#how-to-run.

I also used VSCode Insider with the latest Omini extension but the result still is the same.

Regards,
Huy

**Comment by GuardRex** [comment]
@Smilefounder That's an old `dotnet cli`. It was downgraded and not kept current because devs were being introduced to the latest stable bits (RC1) at the Build conference, and they wanted devs to have a good stable experience. However, the cli-samples are in RC2-nightly form, so you will need the latest `dotnet cli` bits from https://github.com/dotnet/cli/tree/rel/1.0.0

Uninstall your current `dotnet cli` (-1598) and download and install the latest `.NET Core SDK Installer` (second column) from the repo for your platform.

Also note that if your VS Code didn't update you, you can go off of the "Insiders Edition" and go with the 1.0 release at https://code.visualstudio.com/

**Comment by Smilefounder** [comment]
Thanks @GuardRex for your support. :100: 

> Uninstall your current dotnet cli (-1598) and download and install the latest .NET Core SDK Installer (second column) from the repo for your platform.
Actually, I did download and install as you mentioned before I posted the issue here.
But before that, I also install the file from (first column) as well, do I need to uninstall anythings before install the (second column)? It doesn't required to, isn't it?

Anyway, look like I have to wait for the more stable RC2 version to try again.

**Comment by GuardRex** [comment]
The first column `.NET Core Installer` only installs the shared framework. I don't think there would be a problem in installing the shared framework with `dotnet cli` tools with `.NET Core SDK Installer ` (second column) over it. I would only have a concern if they had different version numbers (which they do right now btw). Safest thing to do is uninstall all of the `dotnet cli` bits via `Programs & Features` and just install the `.NET Core SDK Installer`.

Nuke your package caches: You can use `nuget locals all -clear` with the `nuget.exe` (see https://docs.nuget.org/consume/command-line-reference for details).

It is (fairly) stable right now ... RC2 I mean. As long as you use the cli-samples with the latest RC2 bits with the right NuGet feeds (CI Release). If you need help walking through any probs, hit us up on JabbR: https://jabbr.net/#/rooms/AspNetCore

**Comment by Smilefounder** [comment]
I see, thanks again for quick support. Let me try again then keep you posted.

Cheer.

**Comment by Smilefounder** [comment]
Thanks for GuardRex recommended, Fixed the issue with https://github.com/dotnet/cli/issues/1919
